session6/src/com/itschool/homework/HomeRunChallenge4.java

```diff
@@ -0,0 +1,14 @@
+package com.itschool.homework;
+
+public class HomeRunChallenge4 {
+    public static void main(String[] args) {
+        HomeRunChallenge4 home = new HomeRunChallenge4();
+        challengeRun4();
```

Comment:
You declared this method static at line 8 and you are also invoking it statically here, so the above `home` object is not used.